jmxtrans-output/jmxtrans-output-logback/src/main/java/com/googlecode/jmxtrans/model/output/NagiosWriter.java

```diff
@@ -55,7 +57,7 @@
  */
 public class NagiosWriter extends BaseOutputWriter {
```

Comment:
Why oh why does the NagiosWriter depends on log4j to do its job... We should really kill that dependency (and not replace it with a dependency on logback / slf4j). (but that might well be outside the scope of this PR).

Reply:
The NagiosWriter doesn't depend on Lo4J anymore.

This jmxtrans-output-module depends on Logback internals to create a RollingFileAppender per metric.

Reply:
I meant "Why oh why does the NagiosWriter depends on the internals of any logging framework". logback / log4j (or other), I don't really care. (But don't spend too much time on that cleanup, it probably belongs to another PR...)

Reply:
The NagiosWriter is vary close to KeyOutWriter, it uses a FileAppender. The only difference is log format.